docs/mkdocs/documentation/module_loader_image.md

```diff
@@ -0,0 +1,96 @@
+# Creating a ModuleLoader image
+
+Kernel Module Management works with purpose-built ModuleLoader images.
+Those are standard OCI images that satisfy a few requirements:
+
+- `.ko` files must be located under `/opt/lib/modules/${KERNEL_VERSION}`
```

Comment:
```suggestion
- `.ko` files must be located under `/opt/lib/modules/${KERNEL_VERSION}`. In case `spec.moduleLoader.container.modprobe.dirname` is different than `/opt` the the `.ko` files should be placed accordigenly.
```
?